pkg/apis/kappctrl/v1alpha1/status.go

```diff
@@ -32,6 +32,8 @@ const (
 
 	Deleting     ConditionType = "Deleting"
 	DeleteFailed ConditionType = "DeleteFailed"
+
+	PackageYanked ConditionType = "PackageYanked"
```

Comment:
it seems that this condition should be defined by a package that defines packaging concepts. apis/kappctrl is all about lower level primitives (ie App).